### DISCUSSION TITLE
About 'kAckTimeout', why it sets 16(ms) as a default value?

### DISCUSSION THREAD

**Discussion by YuhangZho** [openthread/openthread#8979] [General]
My question is why  'kAckTimeout' is set 16 as default value?
Because I could not find any description about this parameter both on Thread 1.3 spec and IEEE-802.15.4-2015, I feel confused why 16(ms) becomes the default?

**Comment by jwhui** [comment]
The `kAckTimeout` value is only used when the radio platform does not implement its on ACK timeout - see `OPENTHREAD_CONFIG_MAC_SOFTWARE_ACK_TIMEOUT_ENABLE`.

When implementing the ACK timeout in SW, we have to account for processing delays that can occur in the system. In general, it is better for the radio to handle the ACK timeout.

**Reply by YuhangZho** [reply]
Thanks for your quick resp!
